scripts/run_ci_tasks.sh

```diff
@@ -33,52 +35,21 @@ while true; do
   shift
 done
 
-# install tools not present on raw machine
-if [ "$BITRISE_IO" = "true" ]; then
-    npm install -g react-native-cli
-fi
-
-# verify react-native CLI is installed
-react-native -v
-
-# set up react-native project
-if [[ "$BITRISE_SOURCE_DIR" != "" ]]; then
-    REPO_PATH="${BITRISE_SOURCE_DIR}"
-else
-    REPO_PATH=`dirname "${0}"`/../
-fi
-
 cd $REPO_PATH
-pwd
+
 npm install
 
+# Tests
+npm run test
+
 # Android
 if $ANDROID ; then
     cd example/android
 
-    # build Android
-    PROJECT_PLATFORM_PATH="$(pwd)"
-
-    # Make sure google-services.json exists
-    GOOGLE_SERVICES_FILE_PATH="${PROJECT_PLATFORM_PATH}/app/google-services.json"
-    if [[ ! -f ${GOOGLE_SERVICES_FILE_PATH} ]]; then
-      if [[ "$GOOGLE_SERVICES_JSON" == "" ]]; then
-        echo "ERROR: You must provide ${GOOGLE_SERVICES_FILE_PATH}."
-        exit 1
-      else
-        echo $GOOGLE_SERVICES_JSON > ${GOOGLE_SERVICES_FILE_PATH}
-      fi
-    fi
-
-    # Make sure airshipconfig.properties exists
-    if [[ ! -f ${PROJECT_PLATFORM_PATH}/app/src/main/assets/airshipconfig.properties ]]; then
-      cp -np ${PROJECT_PLATFORM_PATH}/app/src/main/assets/airshipconfig.properties.sample ${PROJECT_PLATFORM_PATH}/app/src/main/assets/airshipconfig.properties || true
-    fi
-
-    # Build sample
+    # Build
     ./gradlew app:assembleDebug
 
-    cd ../../
+    cd -
```

Comment:
I had to look this command up. For those unfamiliar, it means "cd to the previous working directory"